src/main/resources/changesets/changelog-consent-2025-12-02-alter-dataset-name-column.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet id="changelog-consent-2025-12-02-alter-dataset-name-column" author="kevinmarete">
+    <comment>
+      Alter name column in dataset table to be not nullable.
+      The name field is a required property in the Dataset model.
+    </comment>
+    <addNotNullConstraint tableName="dataset" columnName="name" columnDataType="VARCHAR(255)"/>
```

Comment:
Will this fail if there are existing null values? Do we need an update for the cases where they are?

Reply:
This was already handled [here](https://broadworkbench.atlassian.net/browse/DT-2225?focusedCommentId=102680), you may have to run this query locally though but in dev and prod we confirmed there are no datasets with null names.